packages/server/src/lib/ab.ts

```diff
@@ -56,3 +62,100 @@ export const selectVariant = <V extends Variant, T extends Test<V>>(test: T, mvt
 
     return selectWithSeed(mvtId, seed, test.variants);
 };
+
+// Amounts test code should match the equivalent code in support-frontend as closely as possible
+const amountsRandomNumber = (mvtId: number, seed: number): number => {
+    const rng = seedrandom(`${mvtId + seed}`);
+    return Math.abs(rng.int32());
+};
+// function randomNumber(mvtId: number, seed: number): number {
+//     const rng = seedrandom(`${mvtId + seed}`);
+//     return Math.abs(rng.int32());
+// }
```

Comment:
The commented out code is from the `support-frontend` repo. I've kept it in for reviewers in case they question how closely this version of the code matches that version.

I'm not happy that we have two versions of the code in different repos. In an ideal world we'd put the code into its own NPM package and make both repos depend on it ... but we're hoping that passing the amounts test/variant name in the CTA URL, which can then bypass variant selection in `support-frontend`, will be a much more robust solution

Reply:
tbf it's doing the same thing as `selectWithSeed` above -
`Math.abs(seedrandom(mvtId + seed).int32())`
Both types of ab tests could share this function instead

Reply:
Removed commented out lines